pkg/ovs/ovsconfig/ovs_client.go

```diff
@@ -694,6 +695,19 @@ func buildPortDataCommon(port, intf map[string]interface{}, portData *OVSPortDat
 	} else { // ofport not assigned by OVS yet
 		portData.OFPort = 0
 	}
+	var macInUse string
```

Comment:
The name may cause confusing, maybe use `macStr`

Reply:
updated.